torch/_dynamo/variables/tensor.py

```diff
@@ -661,10 +661,12 @@ def method_type(self, dtype=None, non_blocking=False, **kwargs):
             tensortype = next(
                 k for k, v in tensortype_to_dtype.items() if self.dtype in v
             )
-            if self.device.type == "cuda":
-                return ConstantVariable.create(f"torch.cuda.{tensortype.__name__}")
-            else:
+            if self.device.type == "cpu":
                 return ConstantVariable.create(f"torch.{tensortype.__name__}")
+            else:
+                return ConstantVariable.create(
+                    f"torch.{self.device.type}.{tensortype.__name__}"
+                )
```

Comment:
nit: how about `_get_device_module()`? but it will throw an error if the module is none

https://github.com/pytorch/pytorch/blob/main/torch/_utils.py#L913-#L920

Reply:
where do you want to use _get_device_module? To my understanding this function returns module, and we need a string here

Reply:
ah you are right!